week1/project/ecommerce/src/ProductList.jsx

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable react/prop-types */
+
+
+const ProductList = ({ products }) => {
+    return (
+        <ul className="products">
+        {products.map((product) => (
+            <li key={product.id} className="product-item">
+                <div className="product">
+                    <img className="product-img"src={product.image} alt={product.name} />
```

Comment:
⚒️ 

Nice job adding the alt as well! a11y is super important.

```suggestion
                    <img className="product-img" src={product.image} alt={product.name} />
```